docs/acme-integration.rst

```diff
@@ -156,28 +156,44 @@ configuration active, it will check for validity of the certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
-Certificate for subdomains excluding the apex domain
-----------------------------------------------------
+Example: Certificate for apex domain and subdomains
```

Comment:
What is this "apex" thing? I never here this,find it confusing and doubt it will ad any relevant information.